llvm-hs/src/LLVM/Internal/FFI/ErrorHandling.cpp

```diff
@@ -1,5 +1,6 @@
 #include "LLVM/Internal/FFI/ErrorHandling.hpp"
 #include <iostream>
+#include <cstdlib>
```

Comment:
After I downgraded GCC from `7.2` to `5.4`, `llvm-hs` doesn't compile without this import.